syntax = "proto3";

// The tweet service definition.
service Tweet {
  // streams tweets
  rpc RequestTweets (TweetRequest) returns (stream TweetReply) {}
}

// The request message
message TweetRequest {
  int32 number_of_tweets = 1;
}

// The response message containing tweet data
message TweetReply {
  string target = 1;
  string id = 2;
  string date = 3;
  string flag = 4;
  string user = 5;
  string text = 6;
}
